vercel.json

```diff
@@ -1788,6 +1788,82 @@
     {
       "source": "/cloud/release-notes/:path*",
       "destination": "https://temporal.io/change-log/product-area/cloud"
+    },
+    {
+      "source": "/dev-guide/typescript/dev-guide/typescript/foundations#register-types",
+      "destination": "/dev-guide/typescript/foundations#register-types"
+    },
```

Comment:
```suggestion
```
This looks like a bad link in the referring doc. I'm strongly inclined not to bother redirecting a link that's always been invalid. Instead, let's fix the link in https://docs.temporal.io/dev-guide/typescript/debugging#workflowactivity-registration-errors (which looks like a work item for me).